### PR TITLE
Update the README with more details.

### PR DESCRIPTION
**Pull request by yallop**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,28 @@
...
+use the constants and types in `Unistd` along with a representation of the
+host system.  The bindings support a more comprehensive range of seek commands
+than the corresponding functions in the standard OCaml `Unix` module.  The
+[`Unistd_unix_lwt`][unistd_unix_lwt] module exports non-blocking versions of
+the functions in `Unistd_unix` based on the [Lwt][lwt] cooperative threading
+library.
+
+**WARNING**: the current implementation is not portable due to `read`,
+`write`, `pread`, and `pwrite` wrappers that assume 64-bit instruction
+pointers.
```
Comment (dsheets) [inline README.md]: I don't think this is true any more?

Comment (yallop) [inline README.md]: It's true for the moment.  [Here's an example](https://github.com/dsheets/ocaml-unix-unistd/blob/4666dba2f43436b55967c37f0aaab4d628bc2db5/unix/unistd_unix.ml#L151).

Comment (dsheets) [inline README.md]: Oh ugh, the bindings are still dynamic?

Comment (yallop) [inline README.md]: Yes, so that they can release the runtime lock.

**Comment by dsheets** [comment]
I guess this should be rebased. I'm not sure why I didn't merge it. Sorry.

**Comment by yallop** [comment]
It's rebased.

**Pull request merged**